### DISCUSSION TITLE
Tool bar item with badge

### DISCUSSION THREAD

**Discussion by joshwl2003** [xamarin/Xamarin.Forms#14002] [Ideas]
It would be great if there was a simple way to create a toolbar item with a badge counter on it, like you see in almost every shopping cart application available. 

Or allow for custom controls to be placed in the tool bar so long as the meet the requirements of the tool bar area in terms of size..

**Comment by EmilAlipiev** [comment]
There is already TitleView. you can use it to implement custom layout. Please ask your question here always, you will get better answers https://docs.microsoft.com/en-us/answers/products/dotnet?WT.mc_id=xamarinforumsredirect-web-xamarin
